samples/bluetooth/peripheral_ancs_client/README.rst

```diff
@@ -0,0 +1,284 @@
+      Category ID: Email
+      Category Cnt:2
+      UID:         67305985
+      Flags:
+       Positive Action
+       Negative Action
+
+Perform a notification action
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The received notification has two flags: Positive Action and Negative Action. This means that you can perform these two actions on the notification.
+
+The following table shows the format of the message that the application must send back to perform a notification action:
+
+   +------------------+---------------+-----------------------------+
+   | Field            | Example value | Interpretation              |
+   +==================+===============+=============================+
+   | Command ID       | 2             | Perform notification action |
+   +------------------+---------------+-----------------------------+
+   | Notification UID | 01 02 03 04   | 67305985 (0x4030201)        |
+   +------------------+---------------+-----------------------------+
+   | Action           | 00            | Positive                    |
+   |                  |               |                             |
+   |                  | 01            | Negative                    |
+   +------------------+---------------+-----------------------------+
+
+1. Press Button 3 to perform a positive action and verify that the ANCS Control Point (0xD8F3) is updated to ``02 01 02 03 04 00``.
+#. You can also press Button 4 and observe that the server receives a negative action ``02 01 02 03 04 01``.
+
+Retrieve notification attributes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following table shows the relevant part of a request to retrieve notification attributes:
+
+   +------------------+---------------+-----------------------------+
+   | Field            | Example value | Interpretation              |
+   +==================+===============+=============================+
+   | Command ID       | 0             | Get notification attributes |
+   +------------------+---------------+-----------------------------+
+   | Notification UID | 01 02 03 04   | 67305985 (0x4030201)        |
+   +------------------+---------------+-----------------------------+
+   | Attribute ID     | 00            | App identifier              |
+   +------------------+---------------+-----------------------------+
+   | Attribute ID     | 01            | Title                       |
+   +------------------+---------------+-----------------------------+
+   | Length           | 20 00         | 0x0020                      |
+   +------------------+---------------+-----------------------------+
+   | Attribute ID     | 03            | Message                     |
+   +------------------+---------------+-----------------------------+
+   | Length           | 20 00         | 0x0020                      |
+   +------------------+---------------+-----------------------------+
+
+Note that the example application will request all existing attribute types, not only a subset.
```

Comment:
Make this a note if you want it to stand out.

Reply:
IMO, this note does not need extra attention.